packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -259,6 +259,9 @@ export const buttonBorderAccentOutlineColor = DesignToken.create<string>(
 export const controlHeight = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.controlHeight)
 ).withDefault('32px');
+export const controlMinWidth = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.controlMinWidth)
+).withDefault('176px');
```

Comment:
We have 3 listbox based controls, select, combobox and rich text mention.

rich text mention is already defined in terms of menuMinWidth which is the same value of 176px.
Instead of adding a new token can we instead keep all the listboxes in sync and use the menuMinWidth token? I think that the idea is that the menu styling and dropdown styling of select / combobox / rich-text are aligned so the use of menuMinWidth seems appropriate?

Reply:
That was my first thought, but Brandon has consistently used `176px` as a minimum width for all controls, not just the dropdowns, so I could see the token being used across all components.

I assume it's a breaking change, should we just rename `menuMinWidth`? In a follow-on PR?

Reply:
Lets use menuMinWidth in this PR and save spreading it to controls and renaming it for future work / a tech debt. 

Most of our controls don't define it (and may not be trivial to, i.e. icon button mode of icon) and some are defining different values.

Reply:
Reminder to update the PR description to reflect that this PR is no longer adding a new token.